101-data-factory-v2-blob-to-blob-copy/azuredeploy.parameters.json

```diff
@@ -2,6 +2,9 @@
   "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
   "contentVersion": "1.0.0.0",
   "parameters": {
+    "dataFactoryName": {
+      "value": "ADFTutorialFactory0928"
```

Comment:
use GEN-UNIQUE

Reply:
@bmoore-msft - I thought about using it, but then customer will have to go through the following steps to find the name of the data factory: 

1. Click the successful deployment message. 
2. Launch resource group.
3. Search for the new data factory from the list of data factories. 

If this is must, I will go back to the previous where the data factory will have a prefix: ADFTutorialFactory. 

Please let me know. Thanks.

Reply:
How can I make customers enter a name for it so that they don't need to go through the blades to find the name of the data factory? Thanks.

Reply:
Thanks. Updated the file. Uploading it now.